pyproject.toml

```diff
@@ -48,7 +48,7 @@ flash-attn = ["flash-attn>=2.8.3"]
 aim = ["aim>=3.19.0,<4.0"]
 mlflow = ["mlflow"]
 clearml = ["clearml==2.0.0"]
-fms-accel = ["fms-acceleration>=0.6"]
+fms-accel = ["fms-acceleration @ git+https://github.com/foundation-model-stack/fms-acceleration.git#subdirectory=plugins/framework"]
```

Comment:
can you change this to reflect the version we are picking finally instead of main.